packages/data-point/lib/reducer-types/reducer-list/factory.test.js

```diff
@@ -105,4 +105,12 @@ describe('factory#create', () => {
     expect(result.reducers[2].type).toBe('ReducerPath')
     expect(result.reducers[3].type).toBe('ReducerFunction')
   })
+
+  test('factory#throw error if reducer is falsy and not a string', () => {
+    expect(() => factory.create(createReducer, false)).toThrow()
+    expect(() => factory.create(createReducer, [false])).toThrow()
+    expect(() => factory.create(createReducer, [0])).toThrow()
+    expect(() => factory.create(createReducer, [undefined])).toThrow()
+    expect(() => factory.create(createReducer, ['$foo.bar', false])).toThrow()
+  })
```

Comment:
Could we please break each of these scenarios into a separate test? It makes it easier to read the tested scenarios at a glance and prints out nicer when `jest` is run with the `--verbose` flag.

For an example, you can take a look at this from #207: https://github.com/raingerber/data-point/blob/e0d3ae139baab9eb40f4591336824b34dd0794f8/packages/data-point/lib/reducer-types/reducer-helpers/utils/index.test.js#L5-L45

Reply:
we should really update the contributing guidelines with the points you brought up here and on my PR #207 
<img width="656" alt="screen shot 2018-02-15 at 11 03 54 am" src="https://user-images.githubusercontent.com/3729628/36266684-1019d768-1240-11e8-970c-d0ddfaf72afa.png">